PiwikTracker/Queue.swift

```diff
@@ -1,7 +1,6 @@
 import Foundation
 
-protocol Queue {
-    associatedtype T
+public protocol Queue {
```

Comment:
The Protocol with Associated Type Queue had the downside of adding a Generic to the Tracker.
```
class Tracker<Q:Queue> {
   let queue: Q
}
```

I did not like it this way. It doesn't describe the Tracker properly. Also in the current state it is not necessary. We only want to queue and dispatch Events for now. So I changed the queue to only work with Events (at least for now).